parm/config/gefs/config.ufs

```diff
@@ -328,7 +328,7 @@ if [[ "${skip_mom6}" == "false" ]]; then
       MOM6_RESTART_SETTING='r'
       MOM6_RIVER_RUNOFF='False'
       eps_imesh="2.5e-1"
-      TOPOEDITS="ufs.topo_edits_011818.nc"
+      TOPOEDITS="topo_edits_011818.nc"
```

Comment:
Should this still include the "ufs." bit, especially after issue #2812 was completed?

Reply:
I think we still need this change.

Reply:
code updated with ufs.topo